qiskit/providers/backend_compat.py

```diff
@@ -60,9 +60,19 @@ def convert_to_target(
     Returns:
         A ``Target`` instance.
     """
-    return _convert_to_target(
-        configuration, properties, defaults, custom_name_mapping, add_delay, filter_faulty
-    )
+    # If a deprecated error is raised during the conversion, we should not return the
+    # deprecation warning to the user,as it is not actionable for them.
+    with warnings.catch_warnings():
```

Comment:
Shouldn't this whole method be deprecated since all the input to it are deprecated?

Reply:
It is, here: https://github.com/Qiskit/qiskit/pull/12915. It is one of the other PRs in the queue for 1.4.